content/en/docs/reference/scheduling/config.md

```diff
@@ -255,7 +255,183 @@ the same configuration parameters (if applicable). This is because the scheduler
 only has one pending pods queue.
 {{< /note >}}
 
-## {{% heading "whatsnext" %}}
+### MultiPoint config
+
+There is an additional field in the profile config, `MultiPoint`, which allows 
+for easily enabling or disabling a plugin across several extension points. The intent 
+of `MultiPoint` config is to simplify the configuration needed for users and administrators 
+when using custom profiles.
+
+Consider a plugin, `MyPlugin`, which implements the `PreScore`, `Score`, `PreFilter`, 
+and `Filter` extension points. To enable `MyPlugin` for all its available extension 
+points, the profile config simply looks like:
```

Comment:
try to [avoid terms like "simply"](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-words-that-assume-a-specific-level-of-understanding)

```suggestion
points, the profile config looks like:
```